fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoanTransactionsApiResourceSwagger.java

```diff
@@ -137,6 +137,8 @@ private GetLoansCurrency() {}
         public LocalDate submittedOnDate;
         @Schema(example = "false")
         public Boolean manuallyReversed;
+        @Schema(example = "false")
```

Comment:
I believe this field is not needed anymore.

Reply:
Done